src/org/usfirst/frc/team166/robot/subsystems/AimShooter.java

```diff
@@ -48,7 +48,7 @@ private void updatePIDConstants() {
 	}
 
 	private double convertAngleToDisplacement(double angle) {
-		double displacement = ((angle - 45) * voltsPerDegree);
+		double displacement = ((angle - 40) * voltsPerDegree);
```

Comment:
What does this `40` mean? Is it the same `40` on line 60? If so, it should have been made a constant above to allow you to only have to change one place.